packages/forms/src/model.ts

```diff
@@ -2406,4 +2414,8 @@ export class FormArray extends AbstractControl {
     control.setParent(this);
     control._registerOnCollectionChange(this._onCollectionChange);
   }
+
+  private _adjustIndex(index: number): number {
```

Comment:
Could you pleas add a quick comment here:
```suggestion
  /**
   * Converts negative indices to a corresponding positive number that represents
   * a position of an element in an array.
   */
  private _adjustIndex(index: number): number {
```